metabase/metabase
# Add field values preview when selecting a card source for a parameter in a dashboard

Epic https://github.com/metabase/metabase/issues/26898

How to verify:
- Create a dashboard with a card and a `text -> dropdown` parameter mapped to a column in the card
- Edit dashboard -> Select parameter -> Click `Values from a model or question`
- Select a saved question and its string column to use for the values
- It should show preview of the values
- It should work for both structured and native queries (added e2e tests for these cases)

<img width="662" alt="Screenshot 2023-01-04 at 16 41 46" src="https://user-images.githubusercontent.com/8542534/210579971-c57fdef0-1f0b-4dd8-8706-ccccc4d23aee.png">
